apps/api/src/routes/v1_permissions_createPermission.error.test.ts

```diff
@@ -39,3 +40,38 @@ describe.each([
     });
   });
 });
+
+test("creating the same permission twice returns conflict", async (t) => {
+  const h = await IntegrationHarness.init(t);
+  const root = await h.createRootKey(["rbac.*.create_permission"]);
+
+  const name = randomUUID();
+  // The First request should succeed
+  // The Second request should fail
+  const expectedStatuses: Record<number, number> = {
```

Comment:
fair enough but I feel like just running the function twice without a loop would be much easier to understand
let's keep it as is though

Reply:
I've added them back for clarity's sake